llvm/include/llvm/BinaryFormat/Minidump.h

```diff
@@ -246,6 +246,8 @@ static_assert(sizeof(Thread) == 48);
 
 struct Exception {
   static constexpr size_t MaxParameters = 15;
+  static constexpr size_t MaxParameterBytes = MaxParameters * sizeof(uint64_t);
+  static const uint32_t LLDB_FLAG = 0x80000000;
```

Comment:
Maybe set to to an ASCII integer like this:
```
static const uint32_t LLDB_FLAG = 'LLDB';
```
Then we check if the `exp_record.ExceptionFlags` is equal to LLDB_FLAG, we are sure this is an LLDB thing. We don't want to load something from a different minidump creator and just assume since the high bit is set that it is a LLDB exeption. Or we can leave `exp_record.ExceptionFlags` alone and encode this 'LLDB' as the first item in the `exp_record.NumberParameters` args and then append anything else we need to encode.